python/src/_wrapper.py

```diff
@@ -438,7 +438,7 @@ def batch_search(
 
 class StaticMemoryIndex:
     def __init__(
-        self, metric: Literal["l2", "mips"], vector_dtype: VectorDType, index_path: str
+        self, metric: Literal["l2", "mips"], vector_dtype: VectorDType, index_path: str, l_build: int, graph_degree: int
```

Comment:
We've been using list_size and max_degree. 

FWIW, I'm not happy with either of them, as they don't actually tell me what they do or are for.  With `list_size` I can actually know it's the size of a list of some sort, but I don't know why or how it's being used.  `l_build` means nothing to me unless I also have the paper open at the same time.  `graph_degree` | `max_degree` have about the same problem.  

Regardless of what we pick, I want us to be consistent, and I want us to be descriptive about intent in our naming, with the code being self descriptive enough it won't need a whitepaper to accompany it for it to make sense.  So, I'm open to anything that might hit all of these!